docs/docs/guides/integrate/tools/apache2.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Apache 2.0
+---
+
+This integration guide shows you the basic OpenID Connect integration with ZITADEL and an Apache 2.0 server.
+
+## Setup PKCE client in ZITADEL
+
+- Go to your organization and setup a new application with the type PKCE
+- When created go to the "Redirect Settings" and *enable Development Mode*
+- Add the Redirect Uri, f.e. `http://localhost:8080/secure/callback`
+- Add the Post Logout Uri, f.e. `http://localhost:8080/index.html`
+![Configuration](/img/guides/integrate/tools/apache-configuration.png)
+![Redirect Settings](/img/guides/integrate/tools/apache-redirect_settings.png)
+
+You can find the url to your discovery endpoint under "URLs":
+![Discovery Endpoint](/img/guides/integrate/tools/apache-urls.png)
+
+## Configure Apache2
+
+### Configure mod_auth_openidc
+
+We use the module module `mod_auth_openidc` in this guide.
+You can find a minimal configuration in the [official documentation](https://github.com/OpenIDC/mod_auth_openidc/wiki#16-what-does-a-minimal-apache-configuration-file-look-like).
+
+The following parameters must be set with the values from ZITADEL.
+
+```yaml
+OIDCProviderMetadataURL https://<your_domain>.zitadel.cloud/.well-known/openid-configuration
+OIDCClientID <client_id, eg 227791....@apache_test>
+# OIDCRedirectURI is a vanity URL that must point to a path protected by this module but must NOT point to any content
+OIDCRedirectURI <redirect_uri, eg http://localhost:8080/secure/callback>
+OIDCCryptoPassphrase <very-secure-phrase>
+
+OIDCScope "openid profile"
+OIDCPKCEMethod S256
+```
+
+With the following parameters
+
+| Parameter | Description | Example value|
+|---|---|---|
+| OIDCProviderMetadataURL | Is the url to the discovery endpoint, which is typically located at {your-domain}/.well-known/openid-configuration| https://<your_domain>.zitadel.cloud/.well-known/openid-configuration |
+| OIDCClientID | Is the ID of the zitadel application. You can find it on the settings page of the application.  | 123456789123@apache_test |
+| OIDCRedirectURI | Users will be redirected to this page after successful login. If you are using localhost make sure to enable development mode in ZITADEL. | http://localhost:8080/secure/callback |
```

Comment:
do we need to mention that, it's not only localhost, but rather any non http*s* endpoint?

Reply:
updated the text.